javascripts/govuk/show-hide-content.js

```diff
@@ -16,7 +16,7 @@
 
     // Escape name attribute for use in DOM selector
     function escapeElementName (str) {
-      var result = str.replace('[', '\\[').replace(']', '\\]')
+      var result = '"' + str.replace('"', '\\"') + '"'
```

Comment:
This could just be 
```
function escapeElementName (str) {
  return '"' + str.replace('"', '\\"') + '"'
}
```

Reply:
might be wrong but won't this just replace one instance? might you need

`str.replace(/"/g, '\\"')`

instead?